test/update_test.rb

```diff
@@ -192,7 +208,9 @@ def default_SetDesktopPattern_stubs
     end
 
     context "when the release info does not match" do
-      it "logs an error" do
+      it "logs info and error" do
+        expect(Yast::Update.log).to receive(:info)
+          .with("Version expected: opensuse-leap-15.0. Backup version: opensuse-tumbleweed-20180911")
```

Comment:
Checking the logged values is a bit fragile, rather test the returned values from methods or passed parameters. If you really need to test the logging then I'd suggest to use `.and_call_original` at the and so the message is really logged. The other critical messages are still logged, with partial mock some messages will be missing and that would make the debugging more difficult.